README.md

```diff
@@ -81,6 +81,9 @@ Check the [Install Guide](GUIDE.md) for installation instructions.
 | CONNECTIVITY_CHECK_IMAGE | connectivity-check image to use, will update assisted-service config map with given value                                           |
 | HARDWARE_INFO_IMAGE      | hardware-info image to use, will update assisted-service config map with given value                                                |
 | INVENTORY_IMAGE          | assisted-service image to be updated in assisted-service config map with given value                                                    |
+| ENABLE_AUTH              | configure assisted-service to authenticate API requests, default: false                                                             |
+| OFFLINE_TOKEN            | token used to fetch JWT tokens for assisted-service authentication                                                                  |
```

Comment:
Maybe add the URL from where to get the token: https://cloud.redhat.com/openshift/token